components/script/dom/servoparser/mod.rs

```diff
@@ -1120,13 +1120,15 @@ fn create_element_for_token(
     } else {
         CustomElementCreationMode::Asynchronous
     };
-    let element = Element::create(name, is, document, creator, creation_mode);
+    let element = Element::create_unfinished(name, is, document, creator, creation_mode);
 
     // Step 8.
     for attr in attrs {
         element.set_attribute_from_parser(attr.name, attr.value, None);
     }
 
+    element.done_creating();
```

Comment:
Is this somewhere in the spec btw?

Reply:
I don't think so but without that the input was revalidating it's input when we set value (with a default min/max), then for min then for max and it was causing troubles. For example, setting no value would set it to 50, then setting a min to 2, a max to 6 would set the value to 6 since 50 > 6 instead of setting it to 4.

Reply:
The same thing has happened to me taking an independent attempt at getting min/max/step working; I'm going to pick up this approach and run with it.